src/kirby/components/tabs/tab-button/tab-button.events.ts

```diff
@@ -0,0 +1 @@
+export const tabClicked = 'tabClicked';
```

Comment:
Consider using present tense as this is consistent with built in event naming
```suggestion
export const tabClick = 'tabClick';
```